applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -105,7 +110,7 @@ def AddVariables(self):
         ## Add base class variables
         super(NavierStokesEmbeddedMonolithicSolver, self).AddVariables()
         ## Add specific variables needed for the embedded solver
-        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_H)           # Nodal volume necessary for refinement
+        # self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_H)           # Nodal volume necessary for refinement
```

Comment:
Do not comment out unused lines, remove them (if you ever need them again, they will be somewhere in the git history)